AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -171,7 +171,7 @@ func addServiceTarget(_ name: String) {
         .target(
             name: name,
             dependencies: serviceTargetDependencies,
-            path: "./Sources/Services/\(name)"
+            path: "./Sources/Services/\(name)/Sources/\(name)"
```

Comment:
Changes in this file are to read source & test files from their new location, which is organized within `/Sources/Services/\(name)/` per a standard Swift package.